tests/Jenkins/utilities.sh

```diff
@@ -332,6 +332,26 @@ function diracInstallCommand(){
 	./dirac-install -r `cat dirac.version` -t server -d
 }
 
+#.............................................................................
+#
+# prepareForServer:
+#
+#   This function gets the DIRAC install script
+#
+#.............................................................................
+
+function prepareForServer(){
+	echo '==> [prepareForServer]'
+
+	#Move to server directory
+	cd $SERVERINSTALLDIR
```

Comment:
What I mean is that navigating, especially without checking the return code, is very dangerous. If, for a reason or another, you can go to the target, you remain where you are, and all the commands are executed there. And this is particularly dangerous when ran in a cron (so you start it '/'), and when the command following a failed 'cd' is 'rm -rf'. This is what TheCleaner was doing. If instead of doing
cd X
rm -rf *

you do
rm -rf X/*
it is much saffer